docker/run.bash

```diff
@@ -40,7 +31,6 @@ docker run -it \
   -v "/etc/localtime:/etc/localtime:ro" \
   -v "/dev/input:/dev/input" \
   --rm \
-  --gpus all \
```

Comment:
after this changes this script started working fine on a Ubuntu 22.04 host

Reply:
I think this is needed to get GPU acceleration when using NVIDIA GPUs.  Maybe leave it as is and add a statement below to remove it if not using a GPU?

Reply:
I'll roll-back. Not sure how to dispatch automatically it for a GPU/noGPU cases. In my environment (VirtualBox) it leads to error:
`docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].`